test/parallel/test-http2-client-socket-operations.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const h2 = require('http2');
+
+const server = h2.createServer();
+server.on('stream', common.mustCall((stream) => {
+    stream.respond();
+    stream.end();
+  })
```

Comment:
Indent will need to be adjusted throughout to pass the linter.

Reply:
Adjusted indentation and confirmed that `make lint-js` passes in new commit